service/surf/apps/users/management/commands/monitor_uptime.py

```diff
@@ -0,0 +1,69 @@
+import logging
+import boto3
+import json
+from datetime import date, timedelta, datetime, time
+from suds.client import Client
+from django.core.management.base import BaseCommand
+
+
+logger = logging.getLogger("service")
+SURFRAPPORTAGE_URL = 'https://rapportage.surfnet.nl:9011/interface.php?wsdl'
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        last_day_of_prev_month = datetime.combine(date.today().replace(day=1) - timedelta(days=1), time.max)
+        start_day_of_prev_month = datetime.combine(
+            date.today().replace(day=1) - timedelta(days=last_day_of_prev_month.day),
+            time.min)
+        uptime = self._fetch_uptime(start_day_of_prev_month, last_day_of_prev_month)
+        period = f"{start_day_of_prev_month.year}-{start_day_of_prev_month.strftime('%m')}"
+        self._send_uptime_to_surfrapportage(uptime, period)
+
+    def _fetch_uptime(self, start_day, end_day):
+        session = boto3.Session(region_name='us-east-1')
```

Comment:
US?

Reply:
Ja, de cloudwatch metrics zitten niet in eu-central-1